examples/pub_verif_metadata.example.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2024 Cloudflare, Inc.
+// Licensed under the Apache-2.0 license found in the LICENSE file or at https://opensource.org/licenses/Apache-2.0
+
+import { generatePrimeSync } from 'node:crypto';
+
+import { Extension, Extensions, TOKEN_TYPES, publicVerif } from '../src/index.js';
+type PartiallyBlindRSAMode = publicVerif.PartiallyBlindRSAMode;
+const {
+    PartiallyBlindRSAMode,
+    ClientWithMetadata,
+    IssuerWithMetadata,
+    OriginWithMetadata,
+    getPublicKeyBytes,
+} = publicVerif;
+
+async function setup(mode: PartiallyBlindRSAMode, extensions: Extensions) {
+    // [ Issuer ] creates a key pair.
+    const keys = await IssuerWithMetadata.generateKey(
+        mode,
+        {
+            modulusLength: 2048,
+            publicExponent: Uint8Array.from([1, 0, 1]),
+        },
+        (length: number) => generatePrimeSync(length, { safe: true, bigint: true }),
+    );
+    const issuer = new IssuerWithMetadata(mode, 'issuer.com', keys.privateKey, keys.publicKey);
+    const pkIssuer = await getPublicKeyBytes(issuer.publicKey);
+
+    // [ Client ] creates a state.
+    const client = new ClientWithMetadata(mode, extensions);
+
+    // [ Origin ] creates a state.
+    const origin = new OriginWithMetadata(mode, extensions, [
+        'origin.example.com',
+        'origin2.example.com',
+    ]);
+
+    return { issuer, client, origin, pkIssuer };
+}
+
+async function rsaVariant(mode: PartiallyBlindRSAMode): Promise<void> {
+    // Protocol Setup
+    //
+    // [ Everybody ] agree to use Public Verifiable Tokens with Metadata.
+    const extensions = new Extensions([new Extension(0x0000, new Uint8Array([1, 2, 3]))]);
```

Comment:
this is just to emphasize what metadata is and its packaging as a extension.
Also, the extension type cannot be 0.

```suggestion
    const metadata = new Uint8Array([1, 2, 3]);
    const extensions = new Extensions([new Extension(0xACDC, metadata)]);
```

Reply:
metadata is both the number and its content I'd say. I renamed the variable `info` instead
It might be worth to add grease reserved numbers to the draft, similar to what the main [Privacy Pass draft provides](https://www.ietf.org/archive/id/draft-ietf-privacypass-auth-scheme-15.html#section-6.2.1).
For now, I'll move `0xACDC` to a dedicated `TEST_EXTENSION_TYPE` to make this clear.